docs/comparison.rst

```diff
@@ -0,0 +1,93 @@
+.. _ebtelplusplus-comparison:
+
+Why ebtelplusplus?
+======================
+
+For various historical reasons, there are multiple software implementations of the EBTEL model.
+There are currently *three* maintained implementations of the model as described below.
+Hereafter, "EBTEL" refers to the model generally while each software implementation is given
+its own unique name (e.g. ``ebtelplusplus``).
+This page briefly explains the differences between each implementation and the advantages of
+the ``ebtelplusplus`` implementation.
+
+The table below summarizes the comparison of the different EBTEL software implementations and the different features provided by each.
+
+.. list-table::
+   :header-rows: 1
+   :align: center
+
+   * - Feature
+     - Citation
+     - |EBTEL-IDL|_
+     - ``ebtel++``
+     - |EBTEL3-IDL|_
+   * - Decouple electrons and ions
+     - :cite:t:`barnes_inference_2016`
+     - :octicon:`x;1em;sd-text-danger`
+     - :octicon:`check;1em;sd-text-success`
+     - :octicon:`x;1em;sd-text-danger`
+   * - Adaptive time step
+     - :cite:t:`barnes_inference_2016`
+     - :octicon:`x;1em;sd-text-danger`
+     - :octicon:`check;1em;sd-text-success`
+     - :octicon:`check;1em;sd-text-success`
+   * - Area expansion
+     - :cite:t:`cargill_static_2022`
+     - :octicon:`check;1em;sd-text-success`
+     - :octicon:`check;1em;sd-text-success`
+     - :octicon:`x;1em;sd-text-danger`
+   * - Supersonic flows
+     - :cite:t:`rajhans_flows_2022`
+     - :octicon:`x;1em;sd-text-danger`
+     - :octicon:`x;1em;sd-text-danger`
+     - :octicon:`check;1em;sd-text-success`
+
```

Comment:
Time-variable abundances!  ebtel++ is currently the only code that does this.  (Working on HYDRAD)

Reply:
How could I forget! 🤦 Added to the table and added a short note in the description as well.